include/pisa/compress.hpp

```diff
@@ -74,16 +75,14 @@ void compress_index(
     {
         pisa::progress progress("Create index", input.size());
         WandType wdata;
-        mio::mmap_source md;
-        if (wand_data_filename) {
-            std::error_code error;
-            md.map(*wand_data_filename, error);
-            if (error) {
-                spdlog::error("error mapping file: {}, exiting...", error.message());
-                std::abort();
+        auto wdata_source = [&] {
```

Comment:
why a lambda here? Do we need to re-use it? 
I guess we could have one `load_wand_file` function and share it everywhere. What do you think?